es6-shim/es6-shim.d.ts

```diff
@@ -480,15 +480,15 @@ interface Promise<T> {
     * @returns A Promise for the completion of which ever callback is executed.
     */
     then<TResult>(onfulfilled?: (value: T) => TResult | PromiseLike<TResult>, onrejected?: (reason: any) => TResult | PromiseLike<TResult>): Promise<TResult>;
-    then<TResult>(onfulfilled?: (value: T) => TResult | PromiseLike<TResult>, onrejected?: (reason: any) => void): Promise<TResult>;
+    then<TResult>(onfulfilled?: (value: T) => TResult | PromiseLike<TResult>, onrejected?: (reason: any) => any): Promise<TResult>;
```

Comment:
I disagree with this change, for the reasons [mentioned above](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9094/files#r60949329).